From eeac8c11c935edf9eea2bed825af6c57e9fb52e3 Mon Sep 17 00:00:00 2001
From: Rich McAllister <Nopublic@address.provided>
Date: Tue, 31 Mar 2020 21:01:21 -0700
Subject: [PATCH] abstractions: add /etc/mdns.allow to  /etc/apparmor.d/abstractions/mdns
References: bsc#1168306

In focal users of mdns get denials in apparmor confined applications.
An exampel can be found in the original bug below.

It seems it is a common pattern, see
https://github.com/lathiat/nss-mdns#etcmdnsallow

Therefore I'm asking to add
   /etc/mdns.allow r,
to the file
   /etc/apparmor.d/abstractions/mdns"
by default.

---
 profiles/apparmor.d/abstractions/mdns |    1 +
 1 file changed, 1 insertion(+)

--- a/profiles/apparmor.d/abstractions/mdns
+++ b/profiles/apparmor.d/abstractions/mdns
@@ -9,5 +9,6 @@
 # ------------------------------------------------------------------
 
   # mdnsd
+  /etc/mdns.allow r,
   /etc/nss_mdns.conf r,
   /{,var/}run/mdnsd w,
